lib/EasyCSV/Reader.php

```diff
@@ -1,52 +1,47 @@
 <?php
 
+declare(strict_types=1);
+
 namespace EasyCSV;
 
+use LogicException;
+use function array_combine;
+use function array_filter;
+use function is_array;
+use function mb_strpos;
+use function sprintf;
+use function str_getcsv;
+use function str_replace;
+
 class Reader extends AbstractBase
 {
-    /**
-     * @var bool
-     */
+    /** @var bool */
     private $headersInFirstRow = true;
 
-    /**
-     * @var array|bool
-     */
+    /** @var string[]|bool */
     private $headers = false;
 
-    /**
-     * @var
-     */
+    /** @var bool */
     private $init;
 
-    /**
-     * @var bool|int
-     */
+    /** @var bool|int */
     private $headerLine = false;
 
-    /**
-     * @var bool|int
-     */
+    /** @var bool|int */
     private $lastLine = false;
 
-    /**
-     * @var bool|int
-     */
+    /** @var bool */
     private $isNeedBOMRemove = true;
 
-    /**
-     * @param $path
-     * @param string $mode
-     * @param bool   $headersInFirstRow
```

Comment:
Can you explain why remove these doc block?

Reply:
Because the type hints in the PHP language define everything needed so the doc blocks are not required.